lib/react_client/bridge.dart

```diff
@@ -42,11 +42,11 @@ abstract class Component2Bridge {
   /// `ReactDartComponentFactoryProxy2`, and not when manually instantiated.
   ///
   /// __For internal/advanced use only.__
-  static Component2Bridge forComponent(Component2 component) => bridgeForComponent[component];
+  static Component2Bridge? forComponent(Component2 component) => bridgeForComponent[component];
 
-  void setState(Component2 component, Map newState, SetStateCallback callback);
-  void setStateWithUpdater(Component2 component, StateUpdaterCallback stateUpdater, SetStateCallback callback);
-  void forceUpdate(Component2 component, SetStateCallback callback);
+  void setState(Component2 component, Map? newState, SetStateCallback? callback);
+  void setStateWithUpdater(Component2 component, StateUpdaterCallback stateUpdater, SetStateCallback? callback);
+  void forceUpdate(Component2 component, SetStateCallback? callback);
```

Comment:
Should the `callback` args in these three be optional?

Reply:
Same question for the impl's below.

Reply:
They could potentially be made optional, but I think they're required to ensure they're always explicitly specified and not lost along the way. Also, making them optional would be a tricky change to make since this class is extended in over_react.

 so I think we should keep them as-is unless there's a strong reason to make them optional